### ISSUE TITLE
-Wincompatible-pointer-types-discards-qualifiers in drivers/gpio/gpio-amd-fch.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#379]
from linux-next:
```
    drivers/gpio/gpio-amd-fch.c:164:49: error: passing 'const struct resource *' to parameter of type 'struct resource *' discards qualifiers [-Werror,-Wincompatible-pointer-types-discards-qualifiers]
```
via kernelCI

**Comment by nickdesaulniers** [comment]
https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/drivers/gpio/gpio-amd-fch.c?id=e09d168f13f0d63df7fe095d52be04c16cbe1cef

**Comment by nickdesaulniers** [comment]
looks patched: https://git.kernel.org/pub/scm/linux/kernel/git/linusw/linux-gpio.git/commit/?id=c0162a49e0a0651625f0c22fd45ece4573143a67

**Comment by nathanchance** [comment]
This was fixed in `next-20190301`: https://git.kernel.org/next/linux-next/c/c0162a49e0a0651625f0c22fd45ece4573143a67
